service/src/main/java/org/jboss/aerogear/unifiedpush/service/impl/ClientInstallationServiceImpl.java

```diff
@@ -83,8 +83,9 @@ public void addInstallation(Variant variant, Installation entity) {
 
     private void mergeCategories(Installation entity) {
```

Comment:
I also think a bit of JavaDoc may help - I doubt that it's straightforward method